### PR TITLE
Anup_Assignment_24_Done_40mins

### PR DESCRIPTION
**Pull request by AnupSJun20**
Please review.
PhoneNumberValidation

### CODE REVIEW

File: src/anupSJun20/PhoneNumberValidation.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+public class PhoneNumberValidation {
+
+	static void findPhoneNumberhavingCharacter(String number) {
+		boolean flag = true;
+		for (int index = 0; index < number.length(); index++) {
+			if (Character.isLetter(number.charAt(index))) {
+				System.out.println("The phone number : " + number + " is invalid ");
+				flag = false;
+			}
```
Comment (harshvegada) [inline src/anupSJun20/PhoneNumberValidation.java:22]: once this if condition pass then need to break loop, do not go for further checking.

File: src/anupSJun20/PhoneNumberValidation.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		}
+	}
+
+	static void verifyPhoneNumber(String number) {
+
+		if (number.length() < 10 || number.length() > 14) {
+			System.out.println("The phone number : " + number + " is invalid ");
+		} else if (number.length() == 14) {
+			if (number.startsWith("+91") && number.charAt(3) == ' ' || number.charAt(4) == '7'
+					|| number.charAt(4) == '8' || number.charAt(4) == '9') {
```
Comment (harshvegada) [inline src/anupSJun20/PhoneNumberValidation.java]: For space you should check contsins(" ") and for numbers checks number.startsWith("7/8/9")

Comment (harshvegada) [reply src/anupSJun20/PhoneNumberValidation.java]: same for other conditions

**Review by harshvegada — COMMENTED** [review]
work on comments

File: src/anupSJun20/PhoneNumberValidation.java (hunk 1 above)
Comment (AnupSJun20) [reply src/anupSJun20/PhoneNumberValidation.java:22]: Added the break statement.

File: src/anupSJun20/PhoneNumberValidation.java (hunk 2 above)
Comment (AnupSJun20) [reply src/anupSJun20/PhoneNumberValidation.java]: Updated as per the comment. Please review.

**Review by harshvegada — APPROVED** [review]
Code looks good to me
Keep it up @AnupSJun20

**Review by harshvegada — APPROVED** [review]
Looks Good Anup

**Pull request merged**